config/data/Contacts.php

```diff
@@ -56,7 +56,7 @@
 $GLOBALS['dataTool']['Contacts']['alt_address_country'] = array('value' => "'USA'");
 $GLOBALS['dataTool']['Contacts']['account_id'] = array('related' => array('module' => 'Accounts'));
 $GLOBALS['dataTool']['Contacts']['portal_name'] = array('incname' => 'contact');
-$GLOBALS['dataTool']['Contacts']['portal_password'] = array('same_sugar_hash' => 'portal_name');
+$GLOBALS['dataTool']['Contacts']['portal_password'] = array('sugar_hash' => "'asdf'");
```

Comment:
@ramanenka Won't this conflict with [the pattern we use in the product for demo data](https://github.com/sugarcrm/Mango/blob/master/sugarcrm/install/populateSeedData.php#L318)? Should the pattern in Mango be updated to this pattern so that when you add tidbit data to the product after installing with demodata there isn't a conflict in pattern of Portal passwords?